packages/hash/integration/src/tests/model/page.model.test.ts

```diff
@@ -15,6 +15,8 @@ const logger = new Logger({
 
 let db: PostgresAdapter;
 
+// recreating DB takes longer than the default 5 seconds.
+jest.setTimeout(60000);
```

Comment:
I think I have a slight preference for sticking this into the `jets.config` using [the `testTimeout` option](https://jestjs.io/docs/configuration#testtimeout-number), just because it means there's one less thing we have remember to configure at the start of each test file.